test_mcp_debug_quantum.py

```
@@ -15,51 +15,67 @@
 # Add project root to path
 sys.path.append(str(Path(__file__).parent))
 
-from connectors.mcp_debug_tool import MCPDebugTool, MCPDebugContext, DebugResponse
+from connectors.mcp_debug_tool import (
+    MCPDebugTool,
+    MCPDebugContext,
+    DebugResponse,
```

Ruff (0.11.9)

`connectors.mcp_debug_tool.DebugResponse` imported but unused

Remove unused import: `connectors.mcp_debug_tool.DebugResponse`

(F401)